src/dispatchers/user/index.ts

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+import {setAuthentication} from 'store/user';
+import {AppDispatch} from 'types/store';
+
+export const authenticateUser = () => async (dispatch: AppDispatch) => {
+  const {data} = await axios.post<any>(`${process.env.REACT_APP_BACKEND_API}/login`);
```

Comment:
`authenticateUser` will take a parameter to post that to backend right ?

Reply:
Yep, sorry, for responding this late, I've missed the comment.
At the moment this was put as just an example until the API is done.